### ISSUE TITLE
Strange 3d artifact is generated at empty scene

### ISSUE THREAD

**Issue by Chaosus** [godotengine/godot#37192]
**Godot version:**
3.2.1

**OS/device including version:**
Win10/GTX1060

**Issue description:**
The spatial editor in editor and game modes contains a strange graphical artifact that could not be selected or removed.

![image](https://user-images.githubusercontent.com/3036176/77186765-72d6f800-6ae4-11ea-8a81-344153321846.png)

**Steps to reproduce:**
Create a simple project - open 3D editor - at the green line you will see it

![image](https://user-images.githubusercontent.com/3036176/77187103-01e41000-6ae5-11ea-86c1-b5ae3e9f204d.png)

**Minimal reproduction project:**
[New Game Project.zip](https://github.com/godotengine/godot/files/4360893/New.Game.Project.zip)

**Comment by bojidar-bg** [comment]
Duplicate of #10204?

**Comment by Chaosus** [comment]
Yeah, seems like.. but that issue is closed and the bug is still present

**Comment by tinmanjuggernaut** [comment]
There's also a dot that appears in the sky at the zenith as noted in #10204.

For Godot 4, I'm wondering if @clayjohn already discovered and fixed it for 4.0 in PR #37179.

**Comment by clayjohn** [comment]
I never noticed the problem. But it is fixed by #37179

**Comment by nathanfranke** [comment]
To reproduce easily, set `background_sky_custom_fov` to a low value such as 1

**Comment by clayjohn** [comment]
I remember discussing this months ago and we thought it was caused by a floating point precision error.

**Comment by lyuma** [comment]
As noted in issue #48010 about a different but related issue in master branch, I suspect the problem is caused by texture repeat settings on the generated procedural texture.

**Comment by Calinou** [comment]
I can confirm this on 3.3.2 (Linux + NVIDIA).

As a workaround, configure the sky basis as follows:

![image](https://user-images.githubusercontent.com/180032/120110775-46e6e600-c16f-11eb-8e9e-b782a8fcb9e7.png)

I found this workaround by pure chance by playing around with the Sky Orientation property… :eyes:
